### PR TITLE
Added contributors to Readme

### PR DESCRIPTION
**Pull request by versavale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,4 +112,8 @@ default: {default value}
 'type' and 'array' key in validation json should be prefixed with '$' symbol
 on missing $ symbol your validation might not work.
 
-```
\ No newline at end of file
+```
+
+# Contributors:
```
Comment (arshadkazmi42) [inline README.md]: Can you remove `:` from this?

Comment (versavale) [reply README.md]: Sure!

**Comment by arshadkazmi42** [comment]
@versavale Looks good. Just a small change.
thank you for working on this.

**Comment by arshadkazmi42** [comment]
@versavale All looks good now.
Congrats on landing your first PR.

1 suggestion, try tagging the issue you are making the patch for in your PR description, its easy to keep track of issues and its PR

**Comment by versavale** [comment]
Great suggestion! Thank you for your patience!

**Pull request merged**